Misc/NEWS.d/next/Library/2021-09-15-23-49-21.bpo-21302.KTsR8i.rst

```diff
@@ -0,0 +1 @@
+In Windows operating system, :func:`time.sleep` now uses the ``waitable timer`` object, which allows to sleep for an interval specified with microsecond precision.
```

Comment:
You can set the desired sleep time value in resolution of 100 ns but in reality Windows is too crappy OS and the real sleeping time will be between milliseconds and microseconds. This is the real timing jitter of Windows.

Reply:
Usually, I only document the "API resolution". Python is just a thin wrapper to OS function. We don't document OS "effective" resolution, since it depends on the OS and the OS version. It would be too much work to update the Python documentation when an OS is updated.

Reply:
This my modification, is it fine?

In Windows operating system, :func:`time.sleep` now uses the ``waitable timer`` object, which allows to sleep for an interval absolute time. Desired sleep time can be set in 100 nsecs resolution.